subdivisionsconst.go

```diff
@@ -5050,12 +5050,36 @@ const (
 	SubdivisionLTUT SubdivisionCode = "LT-UT"
 	// SubdivisionLTVL      SubdivisionCode = "LT-VL"
 	SubdivisionLTVL SubdivisionCode = "LT-VL"
-	// SubdivisionLUD       SubdivisionCode = "LU-D"
+	// SubdivisionLUD       SubdivisionCode = "LU-D" (deprecated)
```

Comment:
Here is the syntax to tag a constant as deprecated, word surrounded by square brackets is the way to link in godoc

```suggestion
	// SubdivisionLUD       SubdivisionCode = "LU-D"
	// Deprecated, replaced by [SubdivisionLUDR]
```